src/Client/Properties/launchSettings.json

```
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:59835/",
+      "applicationUrl": "https://localhost:59835/",
       "sslPort": 0
```

```suggestion
      "sslPort": 59835
```
I get connection refused over https if sslPort isn't set. Alternatively we could remove the IIS Express profile completely

---

I'm all for removing it completely